<launch>
  <arg name="device" default=""/>
  <arg name="ns" default="audio"/>
  <arg name="dst" default="alsasink"/>
  <arg name="do_timestamp" default="false"/>
  <arg name="format" default="wave"/>
  <arg name="channels" default="1"/>
  <arg name="sample_rate" default="44100"/>
  <arg name="sample_format" default="S16LE"/>
  <arg name="audio_topic" default="audio" />

  <group>
    <node name="audio_play_node" pkg="audio_play" exec="audio_play_node" output="screen">
      <param name="device" value="$(var device)"/>
      <param name="dst" value="$(var dst)"/>
      <param name="do_timestamp" value="$(var do_timestamp)"/>
      <param name="format" value="$(var format)"/>
      <param name="channels" value="$(var channels)"/>
      <param name="sample_rate" value="$(var sample_rate)"/>
      <param name="sample_format" value="$(var sample_format)"/>
      <remap from="audio" to="$(var audio_topic)" />
    </node>
  </group>
</launch>
